github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -246,8 +246,8 @@ presubmits:
           requests:
             memory: "8Gi"
   - name: check-provision-k8s-1.17
-    always_run: true
-    optional: false
+    always_run: false
+    optional: true
```

Comment:
it won't trigger the reconcile loop bug right ?

Reply:
As we don't use the config-forker here I believe we won't run into this situation, no.